testing/dart/canvas_test.dart

```diff
@@ -269,64 +269,4 @@ void main() {
     expectArgumentError(() => canvas.drawRawAtlas(image, Float32List(0), Float32List(4), null, null, rect, paint));
     expectArgumentError(() => canvas.drawRawAtlas(image, Float32List(4), Float32List(4), Int32List(2), BlendMode.src, rect, paint));
   });
-
-  test('Vertex buffer size reflected in picture size for drawVertices', () async {
-    final PictureRecorder recorder = PictureRecorder();
-    final Canvas canvas = Canvas(recorder);
-
-    const int uint16max = 65535;
-
-    final Int32List colors = Int32List(uint16max);
-    final Float32List coords = Float32List(uint16max * 2);
-    final Uint16List indices = Uint16List(uint16max);
-    final Float32List positions = Float32List(uint16max * 2);
-    colors[0] = const Color(0xFFFF0000).value;
-    colors[1] = const Color(0xFF00FF00).value;
-    colors[2] = const Color(0xFF0000FF).value;
-    colors[3] = const Color(0xFF00FFFF).value;
-    indices[1] = indices[3] = 1;
-    indices[2] = indices[5] = 3;
-    indices[4] = 2;
-    positions[2] = positions[4] = positions[5] = positions[7] = 250.0;
-
-    final Vertices vertices = Vertices.raw(
-      VertexMode.triangles,
-      positions,
-      textureCoordinates: coords,
-      colors: colors,
-      indices: indices,
-    );
-    canvas.drawVertices(vertices, BlendMode.src, Paint());
-    final Picture picture = recorder.endRecording();
-
-
-    const int minimumExpected = uint16max * 4;
-    expect(picture.approximateBytesUsed, greaterThan(minimumExpected));
-
-    final PictureRecorder recorder2 = PictureRecorder();
-    final Canvas canvas2 = Canvas(recorder2);
-    canvas2.drawPicture(picture);
-    final Picture picture2 = recorder2.endRecording();
-
-    expect(picture2.approximateBytesUsed, greaterThan(minimumExpected));
```

Comment:
Would any clients of this API have depended on the old way this was calculated?

Reply:
It is remotely possible but I strongly doubt it. We should instead find a better way to express this -if they were depending on it, they were probably incorrect, e.g. they were assuming that by disposing the picture they'd be disposing of all that memory which is not necessarily the case.

Reply:
Moved this to https://github.com/flutter/engine/pull/20950